.travis.yml

```
@@ -31,4 +31,6 @@ jobs:
       name: Build and publish EON docker image
       script: ci/docker.sh
       if: tag IS present
-      env: DOCKER_IMAGE_NAME='evmapp'
\ No newline at end of file
+      env:
+        - EVMAPP_DOCKER_IMAGE_NAME='evmapp'
+        - BOOTSTRAPTOOL_DOCKER_IMAGE_NAME='bootstraptool'
```

```suggestion
        - BOOTSTRAPTOOL_DOCKER_IMAGE_NAME='evmapp-bootstraptool'
```
To avoid confusion with other tools from SDK for e.g. utxo model.